docs/tutorial/tutorial-0.rst

```diff
@@ -77,6 +77,10 @@ button with have a margin of 50 pixels on each side. If we wanted to define a
 margin of 20 pixels on top of the button, we could have defined ``margin_top=20``,
 or we could have specified the ``margin=(20, 50, 50, 50)``.
 
+	button.style.flex = 1
+
+The flex property sets the flexible length on flexible items. If the element is not flexible, the flex property has no use. In this case, it gives a flexibility of 1 to the button created. For more information on this check out https://css-tricks.com/almanac/properties/f/flex/
```

Comment:
Referencing CSS documentation isn't a great idea here - this isn't CSS, and although the semantics are almost identical, referencing CSS documentation (and unofficial documentation at that) might give the impression that any CSS directive is valid.

Reply:
@freakboy3742 can you tell some place to read about flex and what I should add here